extensions/arc/package.nls.json

```diff
@@ -98,12 +98,23 @@
 	"arc.postgres.new.dialog.title": "Deploy a PostgreSQL server group on Azure Arc (preview)",
 	"arc.postgres.settings.section.title": "PostgreSQL server group settings",
 	"arc.postgres.settings.resource.title": "PostgreSQL server group resource settings",
+	"arc.postgres.settings.storage.title": "PostgreSQL server group storage settings",
 	"arc.postgres.server.group.name": "Server group name",
 	"arc.postgres.server.group.name.validation.description": "Server group name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, and be 10 characters or fewer in length.",
 	"arc.postgres.server.group.workers": "Number of workers",
 	"arc.postgres.server.group.port": "Port",
+	"arc.postgres.server.group.engine.version": "Engine Version",
+	"arc.postgres.server.group.extensions.label": "Extensions",
+	"arc.postgres.server.group.extensions.description": "A comma-separated list of the Postgres extensions that should be loaded on startup. Please refer to the postgres documentation for supported values.",
```

Comment:
Should Postgres be capitalized here?

Reply:
It is capitalized in arguments.py for the help of the extension and engine version options. However it is not consistent throughout the file.